index.js

```diff
@@ -462,67 +398,14 @@ let Fixtures = {
      * @return {ChakramPromise} The result of the request to link the records.
      */
     link(left, linkName, right) {
-        let url = utils.constructUrl(VERSION, left._module, left.id, 'link');
-        let params = {headers: this._headers};
+        let url = [left._module, left.id, 'link'].join('/');
+        let params = { };
```

Comment:
Remove space between an empty object? In fact, the params argument is optional for the "post" per the doc of Chakram, and we don't modify it anywhere. @rhoggSugarcrm said we are award of this.